### PR TITLE
Add About component

### PR DESCRIPTION
**Pull request by herecomesjaycee**

### CODE REVIEW

**Comment by herecomesjaycee** [comment]
Screenshot 2019-10-20 at 11 19 12
Screenshot 2019-10-20 at 11 19 56
@GenesisSelf could you do a quick sense check on the styling on this page? It'd be interesting to see if there is any solution outside where the header and footer will not block the content at the top/bottom of the page after scrolling

File: src/App.js [hunk 1]
```diff
@@ -19,36 +20,7 @@ function App() {
   return (
     <div className="App">
       <header className="App-header sticky" id="header">
-        <ul>
```
Comment (GenesisSelf) [inline src/App.js]: Why is this being deleted?

File: src/components/about/about.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+.top-text{
+  padding: 1rem 2rem;
+}
+
+
+.bottom-section {
+  background-color: #D81921;
+}
+
+.person {
```
Comment (GenesisSelf) [inline src/components/about/about.css]: Can we rename this padding to `padding-left-1` or in some fashion that could be then be reusable?

File: src/components/about/svg/about.svg [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<svg width="150" height="250" viewBox="0 0 197 300" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (GenesisSelf) [inline src/components/about/svg/about.svg]: Thanks for this.

File: src/components/about/svg/about.svg [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M127.02 13.3569C126.691 13.0178 126.002 12.3043 125.634 12.014C125.547 11.9446 124.99 11.2688 124.969 11.27C124.4 11.3065 122.714 11.5521 122.434 12.0789C122.402 12.1161 122.372 12.1507 122.343 12.1805C122.025 12.5123 121.721 12.8586 121.425 13.2121C120.776 13.9882 120.172 14.8193 119.586 15.6474C118.445 17.2601 117.38 18.9378 116.36 20.6319C115.321 22.3565 114.357 24.1246 113.343 25.8641C111.884 28.3671 110.237 30.749 108.783 33.2522C107.855 34.8506 107.128 36.5245 106.774 38.3304C106.769 38.3521 106.765 38.3733 106.761 38.3955C106.759 38.4045 106.758 38.4136 106.756 38.4227C106.704 38.7008 106.66 38.982 106.626 39.2667C106.598 39.5103 106.569 39.7548 106.543 39.9998C107.566 38.8502 108.467 37.5232 109.38 36.2789C109.469 36.1571 109.556 36.0336 109.642 35.9094C109.892 35.6743 110.144 35.4422 110.399 35.2116C112.002 33.7578 113.608 32.3116 115.173 30.8136C117.988 28.1193 120.804 25.3522 123.91 23.0175C124.828 22.3281 125.758 21.6563 126.668 20.9574C127.187 20.5598 127.697 20.151 128.198 19.73C128.556 19.4289 130.027 18.4391 129.828 17.5548C129.726 17.1024 127.875 14.2367 127.02 13.3569Z" fill="#2C2C2C"/>
```
Comment (GenesisSelf) [inline src/components/about/svg/about.svg]: This line is coming up as a different colour and not sure what could be causing it..

File: src/index.js [hunk 5]
```diff
@@ -4,12 +4,14 @@ import './index.css';
...
 import * as serviceWorker from './serviceWorker';
 import { BrowserRouter,Route } from 'react-router-dom';
+import About from './components/about/about'
 import Results from './containers/results';
 
 ReactDOM.render(
   <BrowserRouter>
     <App />
-    <Route path="/venues" component={Results} />	
+    <Route path="/about" component={About} />
```
Comment (GenesisSelf) [inline src/index.js]: Nice!

File: src/App.js (hunk 1 above)
Comment (herecomesjaycee) [reply src/App.js]: Oops will revert this, this was deleted coz this block was obstructing the view of other pages during my build stage.

File: src/App.js [hunk 6]
```diff
@@ -20,7 +19,30 @@ function App() {
...
+        <ul>
+          <li>
+            <a>Period Friendly Bristol</a>
+          </li>
+          {/* <li>
+            <img src={logo} className="App-logo mg-left-2" alt="logo" />
+          </li> */}
+        </ul>
       </header>
+      <div className="container-middle">
```
Comment (GenesisSelf) [inline src/App.js]: Is there anyway this about page can updated locally with what is current on `master`?

**Comment by GenesisSelf** [comment]
> Screenshot 2019-10-20 at 11 19 12
> Screenshot 2019-10-20 at 11 19 56
> @GenesisSelf could you do a quick sense check on the styling on this page? It'd be interesting to see if there is any solution outside where the header and footer will not block the content at the top/bottom of the page after scrolling

Maybe we can include and then in a separate PR I can address how it works with all the screens?

File: src/App.js (hunk 6 above)
Comment (GenesisSelf) [reply src/App.js]: Nvm this is outdated.

File: src/components/about/about.svg [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<svg width="249" height="350" viewBox="0 0 249 350" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (GenesisSelf) [inline src/components/about/about.svg:1]: I am happy to move these SVGs to the assets folder I created whenever I make my next PR. I am happy for this PR to be merged if you are :)

**Comment by herecomesjaycee** [comment]
@GenesisSelf thank you! Merging

Just to clarify, we are not doing svg in individual components anymore? I'm happy either way, will need to update README so Jonah is aware.

**Pull request merged**